lib/mixin/sentry_state_transaction_mixin.dart

```diff
@@ -0,0 +1,21 @@
+library after_layout;
+
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+mixin SentryStateTransactionMixin<T extends StatefulWidget> on State<T> {
+  ISentrySpan? transaction;
+
+  @override
+  void initState() {
+    super.initState();
+    transaction = Sentry.startTransaction(
+      runtimeType.toString(),
+      'ui.load',
+      bindToScope: true,
+    );
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
+      transaction?.finish();
```

Comment:
you can use `SpanStatus.ok` also for all the other spans/transactions when there's no errors

Reply:
Is there no default?

Reply:
no, default is unknown by the server or UI